quodlibet/quodlibet/qltk/pluginwin.py

```diff
@@ -154,20 +206,31 @@ def cell_data(col, render, model, iter_, data):
         self.append_column(column)
 
         render = Gtk.CellRendererPixbuf()
+        render.set_padding(2, 2)
 
         def cell_data2(col, render, model, iter_, data):
             plugin = model.get_value(iter_)
             icon = plugin.icon or Icons.SYSTEM_RUN
             render.set_property('icon-name', icon)
 
+            theme = Gtk.IconTheme.get_default()
+            width = height = 24
+            icon_info = theme.lookup_icon(icon, max(width, height), 0)
+            filename = icon_info.get_filename()
+            pixbuf = GdkPixbuf.Pixbuf.new_from_file_at_size(filename,
+                                                            width, height)
+            surface = get_surface_for_pixbuf(self, pixbuf)
+            render.set_property("surface", surface)
```

Comment:
scratch all that and just use ``render.set_property('stock-size', Gtk.IconSize.LARGE_TOOLBAR)``